SU2_CFD/include/solvers/CIncEulerSolver.hpp

```diff
@@ -385,4 +400,22 @@ class CIncEulerSolver : public CFVMFlowSolverBase<CIncEulerVariable, INCOMPRESSI
    */
   inline bool GetHasHybridParallel() const final { return true; }
 
+  /*!
+   * \brief Get the massflow of the streamwise periodic donor/outlet boundary.
+   * \return The streamwise periodic donor/outlet massflow.
+   */
+  su2double GetStreamwise_Periodic_MassFlow() const { return Streamwise_Periodic_MassFlow; }
```

Comment:
`override` on these three

Reply:
Sorry, `final` since they are in the EulerSolver

Reply:
I am favouring (to cite you) `make a type for streamwise periodic things [...]these 3 go everywhere together ` and extending that to 4 with the one that is plugged through the conftig rn. So this method will probably disappear anyway, but i'll keep that in mind for the replacement 👍